### DISCUSSION TITLE
All namespaces

### DISCUSSION THREAD

**Discussion by elmasoki** [argoproj-labs/argocd-operator#657] [Q&A]
Hi,

We installed ArgoCD  on Openshift using ArgoCD operator.  When we change  namespace  to " All Namespace" for default  kubernetes cluster It turns to argocd namespace automatically.  we had to stop [argocd-operator-controller-manager] pod to keep namespace "All namespace". 

How could we keep it as "all namespace"  without stopping [argocd-operator-controller-manager] . 

thank you in advance.

![image](https://user-images.githubusercontent.com/10141608/168089429-19cb0d7b-b7c5-402f-a3fa-54b417f5a479.png)

**Comment by iamraj007** [comment]
Interesting scenario, would love to know how to..

**Comment by jujubetsz** [comment]
You can add this env var in the Operator NS:

```
- name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
  value: '*'
```

Ref: https://argocd-operator.readthedocs.io/en/latest/usage/basics/#cluster-scoped-instance

After re-deploy of argocd instance, the option `NAMESPACES` is set to `All namespaces`
